src/main/java/org/s1ck/gdl/GDLLoader.java

```diff
@@ -163,13 +183,25 @@ public void enterGraph(GDLParser.GraphContext graphContext) {
       graphs.add(g);
     }
     currentGraphId = g.getId();
+    lastSeenGraph = g;
   }
 
   @Override
   public void exitGraph(GDLParser.GraphContext ctx) {
     inGraph = false;
   }
 
+
+  @Override
```

Comment:
public methods should have comments
and all public methods are defined before the private methods inside the class (or at least they should be)